src/sql/base/browser/ui/table/plugins/checkboxSelectColumn.plugin.ts

```diff
@@ -33,24 +33,31 @@ export interface ICheckboxCellActionEventArgs {
 	column: number;
 }
 
+interface ICheckboxColumnValue {
+	enabled: boolean;
+	checked: boolean;
+}
+
+const HeaderCheckboxTitle: string = nls.localize('selectDeselectAll', "Select/Deselect All");
+
 const defaultOptions: ICheckboxSelectColumnOptions = {
 	columnId: '_checkbox_selector',
 	cssClass: undefined,
 	headerCssClass: undefined,
-	toolTip: nls.localize('selectDeselectAll', "Select/Deselect All"),
+	toolTip: undefined,
```

Comment:
I dont know if adding tool tip string is related to accessibility or not - but I think it might be not announcing without it - pease do a check for that.

Reply:
Here I am only updating the default tooltip, it is not appropriate anyway, I have moved this string to a separate const for the checkbox header in this PR.